mathics_scanner/data/named-characters.yml

```diff
@@ -404,6 +406,7 @@ BeamedSixteenthNote:
   wl-unicode-name: BEAMED SIXTEENTH NOTES
 
 Because:
+  amslatex: '\because'
```

Comment:
Don't we need two backslashes because one is escapled?

Reply:
It might be a good idea to add a pytest to make sure that the amslatex field contains only ASCII characters.

Reply:
If one backslash is working, then in the code we probably have one place where we are reading raw and another where we don't.  We should make it so that named-characters.yml's "amslatex" field is consistent, either all escaped or non-escaped. These changes introduce a mixture.

Reply:
In master there is a mixture of both uses. I can see to make the use more homogeneous.

Reply:
On the other hand, using single quotes `'` seems to take the cointain literally, so escaping is not needed.

Reply:
> It might be a good idea to add a pytest to make sure that the amslatex field contains only ASCII characters.

This is done in https://github.com/Mathics3/mathics-scanner/pull/117